PlayTools/MysticRunes/PlayShadow.m

```diff
@@ -0,0 +1,316 @@
+                        method_getImplementation(swizzledMethod),
+                        method_getTypeEncoding(swizzledMethod)) ) {
+        class_replaceMethod(cls,
+                            newSelector,
+                            method_getImplementation(originalMethod),
+                            method_getTypeEncoding(originalMethod));
+    } else {
+        class_replaceMethod(cls,
+                            newSelector,
+                            class_replaceMethod(cls,
+                                                origSelector,
+                                                method_getImplementation(swizzledMethod),
+                                                method_getTypeEncoding(swizzledMethod)),
+                            method_getTypeEncoding(originalMethod));
+    }
+}
+
+// Instance methods
+
+- (NSInteger) pm_hook_deviceType {
+    return 1;
+}
+
+- (bool) pm_return_false {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return false;
+}
+
+- (bool) pm_return_true {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return true;
+}
+
+- (BOOL) pm_return_yes {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return YES;
+}
+
+- (BOOL) pm_return_no {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return NO;
+}
+
+- (int) pm_return_0 {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return 0;
+}
+
+- (int) pm_return_1 {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return 1;
+}
+
+- (NSString *) pm_return_empty {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return @"";
+}
+
+- (NSDictionary *) pm_return_empty_dictionary {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return @{};
+}
+
+// Class methods
+
++ (void) pm_return_2_with_completion_handler:(void (^)(NSInteger))completionHandler {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    completionHandler(2);
+}
+
++ (NSInteger) pm_return_2 {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return 2;
+}
+
++ (bool) pm_clsm_return_false {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return false;
+}
+
++ (bool) pm_clsm_return_true {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return true;
+}
+
++ (BOOL) pm_clsm_return_yes {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return YES;
+}
+
++ (BOOL) pm_clsm_return_no {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return NO;
+}
+
++ (int) pm_clsm_do_nothing_with_callback:(void (^)(int))callback {
+    // NSLog(@"PC-DEBUG: [PlayMask] Jailbreak Detection Attempted");
+    return 0;
+}
+
+@end
+
+@implementation PlayShadowLoader
+
++ (void) load {
+    [self debugLogger:@"PlayShadow is now loading"];
```

Comment:
If you're gonna change it to `PlayMask` please change it everywhere including file name

Reply:
Uh no.